### PR TITLE
Remove Standard environment extender

### PR DESCRIPTION
**Pull request by wendigo**
By default, all environments are now running with a single coordinator and at least one worker.

### CODE REVIEW

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/MultinodeProvider.java [hunk 1]
```diff
@@ -95,56 +79,85 @@ public Standard(
...
-                        .withCopyFileToContainer(forHostPath(dockerFiles.getDockerFilesHostPath("common/standard/config.properties")), CONTAINER_PRESTO_CONFIG_PROPERTIES);
-
-        portBinder.exposePort(container, 8080); // Trino default port
-        return container;
+        return multipleWorkers(1);
     }
 
-    @SuppressWarnings("resource")
-    private DockerContainer createTestsContainer()
+    public EnvironmentExtender multipleWorkers(int workersCount)
```
Comment (leveyja) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/MultinodeProvider.java:99]: nit: might be better to call this `workers(int count)` - `multipleWorkers(1)` seems wrong, but `workers(1)` is correct.
(Also, should you validate `workers > 0`?)

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeKafkaSaslPlaintext.java [hunk 2]
```diff
@@ -40,38 +38,37 @@
     private final ResourceProvider configDir;
 
     @Inject
-    public EnvMultinodeKafkaSaslPlaintext(KafkaSaslPlaintext kafka, StandardMultinode standardMultinode, DockerFiles dockerFiles)
+    public EnvMultinodeKafkaSaslPlaintext(KafkaSaslPlaintext kafka, MultinodeProvider multinodeProvider, DockerFiles dockerFiles)
     {
-        super(standardMultinode, kafka);
+        super(multinodeProvider.singleWorker(), kafka);
         requireNonNull(dockerFiles, "dockerFiles is null");
         configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment/multinode-kafka-sasl-plaintext/");
```
Comment (leveyja) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeKafkaSaslPlaintext.java:45]: It would be nice if we always or never used this - `configDir` is (sometimes) setup to reference the "root" of the particular Environment's configuration, so the `extendEnvironment` call can simply say `configDir.getPath(config.properties)` without having to specify the full path each time.

**Review by leveyja — APPROVED** [review]
Looks good, a few minor suggestions.
I wonder do "EnvSingleNode" and "EnvMultiNode" still make sense, now that all tests are (potentially) multinode?

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/MultinodeProvider.java (hunk 1 above)
Comment (wendigo) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/MultinodeProvider.java:99]: I'll add validation whether count > 1

**Review by nineinchnick — APPROVED** [review]

File: testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop-kerberos/coordinator/config.properties [hunk 3]
```diff
@@ -3,9 +3,9 @@ node.environment=test
 node.internal-address-source=FQDN
 
 coordinator=true
-node-scheduler.include-coordinator=true
+experimental.concurrent-startup=true
```
Comment (nineinchnick) [inline testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/common/hadoop-kerberos/coordinator/config.properties:6]: This is new, is it required? Should it be in a separate commit?

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/Environment.java [hunk 4]
```diff
@@ -685,7 +706,7 @@ private static Consumer<OutputFrame> combineConsumers(Consumer<OutputFrame>... c
 
         private void setContainerOutputConsumer(Function<DockerContainer, Consumer<OutputFrame>> consumer)
         {
-            configureContainers(container -> container.withLogConsumer(consumer.apply(container)));
+            configureContainers(dockerContainer -> true, container -> container.withLogConsumer(consumer.apply(container)));
```
Comment (nineinchnick) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/Environment.java:709]: nit: There are only two occurrences where the predicate is `true` but maybe add an override for this to make this a little bit easier to read.

**Review by nineinchnick — APPROVED** [review]
Nice work, especially given the number of changes required!

**Comment by wendigo** [comment]
Requires https://github.com/trinodb/docker-images/pull/132/files to provide principals for all nodes (including workers)
